pkg/core/system.go

```diff
@@ -21,12 +21,11 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/pivotal/go-ape"
```

Comment:
I find it odd that we can change the import and everything else continues to work. Also it's common to separate core and third party import groups. I'd expect this line to be down by line 31.

^ and for every other file modified.

Reply:
The package name was not changed when the code was extracted, so everything continues to work. 

I will group the imports.